File: tests/fixtures/languages/Sample.java
// Minimal Java fixture for language parse-validation tests.

package demo;

import java.util.List;

public class Sample {
    private final String label;

    public Sample(String label) {
        this.label = label;
    }

    public String greet() {
        return "Hello, " + label;
    }

    public static int add(int a, int b) {
        return a + b;
    }
}

interface Doubler {
    int doubled();
}

File: tests/fixtures/languages/sample.scala
// Minimal Scala fixture for language parse-validation tests.

package demo

import scala.collection.mutable.Map

trait Doubler {
  def doubled: Int
}

class Greeter(label: String) {
  def greet(): String = s"Hello, $label"
}

object Util {
  def add(a: Int, b: Int): Int = a + b
}
